frontend/src/metabase/css/dashboard.css

```diff
@@ -156,9 +156,9 @@
 }
 
 .DashCard .Card {
-  box-shadow: 0px 1px 3px var(--color-shadow);
-  border-color: transparent;
+  border: none;
```

Comment:
Removed `border` completely instead of hiding it using `transparent.
It makes the inside of the card the full width of it rather than subtracting the border width (2px).